plugins/language_haxe.lua

```diff
@@ -0,0 +1,91 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "Haxe Compiler Arguments",
+  files = "%.hxml$",
+  patterns = {
+    { pattern = "[%-][%-%w_]*",      type="keyword"},
+    { pattern = "%.()%u[%w_]*",        type = {"normal", "keyword2"}},
+  },
+  symbols = {}
+}
+
+syntax.add {
+  name = "Haxe String Interpolation",
+  files = "%.hx__string_interp$",
+  patterns = {
+    { pattern = {"%${", "}", "\\"},     type="keyword", syntax = ".hx" },
+    { pattern = {"%$", "%s", "\\"},     type="keyword", syntax = ".hx" },
+    { pattern = "[^ ]",                 type = "string"}
+  },
+  symbols = {}
+}
+
+syntax.add {
+  name = "Haxe",
+  files = "%.hx$",
+  comment = "//",
+  patterns = {
+    { pattern = "%.%.%.",                          type = "operator"   },
+    { pattern = "%<()%u[%w_]*()%>*",               type = {"operator", "keyword2", "operator"}   },
+    { pattern = "%#%s*[%a_]*().*\n",               type = {"keyword", "normal"}   },
+    { pattern = "import%s+()%u[%w]*",              type = {"keyword2", "keyword2"}},
+    { pattern = "import%s+()[%w%.]*%.()%u[%w]*",   type = {"keyword2", "normal", "keyword2"}},
+    { pattern = "//.-\n",                          type = "comment"  },
```

Comment:
`//.*` would be better.

Reply:
Fixed that, though that was just left over from the C# highlighter which I used as a starting point